allennlp/training/metrics/metric.py

```diff
@@ -50,9 +51,11 @@ def from_params(cls, params: Params, vocab: Optional[Vocabulary] = None):
     @staticmethod
     def unwrap_to_tensors(*tensors):
```

Comment:
Can we give a type annotation here now?  Not sure how that works with `*args`, though...